<!-- YAML
added: v0.9.4
-->

* {Error}

当流因底层内部出错而不能产生数据、或推送无效的数据块时触发。

